src/App/components/InputGroup/index.js

```diff
@@ -98,7 +94,10 @@ const InputGroup = ({
                                 ))}{"\n\t"}
                             </select>
                             :
-                            <input {...attrs} />}
+                            <>
+                                <input {...attrs} />
+                                {postfix ? <Addon type={type} value={postfixValue} disabled={disabled} postfix={postfix} /> : null }{prefixValue ? "\n" : null}
```

Comment:
@eskilsen ,
it seems like the bug got introduced between release 10.0.0 & 10.1.0 , which means it most likely comes from this commit: https://github.com/SwedbankPay/design.swedbankpay.com/commit/d4624895ca1f14442c4cc94640525b73309f45ee by its removal of line 103

Now, from my **very** limited knowledge of the codebase 😅 , I'm seeing we're bringing back the `<Addon ...>`  for postfix, but a _slightly_ different version of it. For instance:
- before the removal it was `type={postfixType}` and it is now `type={type}`
- `color` attribute is also removed

Basically:

**before**: `{postfixValue ? <Addon type={postfixType} value={postfixValue} color={postfixBtnColor} disabled={disabled} postfix={postfix} /> : null }{postfixValue ? "\n" : null}</>}`

**after**:  `{postfix ? <Addon type={type} value={postfixValue} disabled={disabled} postfix={postfix} /> : null }{prefixValue ? "\n" : null}`

I have no clue what is right or wrong, and no clue what is this `type` doing yet 😅 , but I just wanted to bring it up to confirm the changes are intentional 👍

Reply:
We can discuss this further in a huddle on Slack 👍